### PR TITLE
added is-really.cool under domains

### PR DESCRIPTION
**Pull request by Joabutt**
### New Submission Check List

Please ensure your submission ticks all of these boxes:

 - [x] This is Software as a Service not self hosted
 - [x] It has a free tier not just a free trial
 - [x] The submission mentions what is free
 - [x] The submission is not already present in the list
 - [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1099,6 +1099,7 @@ Table of Contents
   * [cluster.ws & wip.la](https://github.com/Olivr/free-domain) — Free cluster.ws and wip.la subdomains for developers.
   * [Dee.al](https://domain.dee.al) — Free .dee.al,.gal.aw,.devz.id instant domain for your blog site & Other, ENJOY
   * [is-an.app](https://github.com/tarampampam/free-domains) - Free .is-an.app and .1bt.uk subdomains (wildcards, A, NS, AAAA, CNAME, TXT records are supported).
+  * [is-really.cool](https://is-really.cool) - Free is-really.cool domain for side projects and personal websites.
```
Comment (ripienaar) [inline README.md:1102]: Sorry to ask this - I havnt asked the others - but these cloudflare hosted domains are literally all the same, it looks like you even copy each others code.

How is this better than others that offer the exact same service via cloudflare? I am not inclined to keep adding these low bar services.

Comment (Joabutt) [reply README.md:1102]: i mean i just wanna share it, i could rewrite the code, but I'm currently working on the backend of things so I'm using the frontend of others first. I will update the frontend part soon.

Comment (ripienaar) [reply README.md:1102]: Once you have some differentiating features feel free to come back, else from now I am not adding any more just cloudflare copy and paste services.

Comment (Joabutt) [reply README.md:1102]: im not using cloudflare lol

Comment (Joabutt) [reply README.md:1102]: alright then, thanks for your time. Will you keep this pr open?

Comment (ripienaar) [reply README.md:1102]: I am not here to argue with you, but lets jus ttake a quick look.

your `subdomains.json` has this 

```
{
  "joabutt": "joabutt.github.io",
  "daniel": "daniel.is-a.dev",
}
```

copied from another repo, your own `is-really.cool` is with cloud flare 

```
is-really.cool.         86228   IN      NS      meiling.ns.cloudflare.com.
is-really.cool.         86228   IN      NS      graham.ns.cloudflare.com.
```

I can only assume this is so incomplete that you're just jumping the gun here.

Are you also the person behind `is-a.dev` ? It looks strikingly similar, and if so then its the same entity offering multiple services so should be listed once.

Comment (Joabutt) [reply README.md:1102]: no im working with one of the owners of is-a.dev, so yea he's helping me with the project

Comment (ripienaar) [reply README.md:1102]: Regardless, what about a service without a single user in its registration at all makes you think its a SaaS worthy of listing? I cant event. 

No please do not come back.

Comment (Joabutt) [reply README.md:1102]: And ive discussed with him, that well use cloduflare for the moment until it reaches the 1000 DNS limit, then well switch to another provider.

Comment (Joabutt) [reply README.md:1102]: So im wondering if we changed up the frontend completely and got users using it can we come back

Comment (ripienaar) [reply README.md:1102]: > And ive discussed with him, that well use cloduflare for the moment until it reaches the 1000 DNS limit, then well switch to another provider.

Yet you replied with "im not using cloudflare lol" when clearly you are.

Please, do not come back.

Comment (Joabutt) [reply README.md:1102]: sorry for wasting your time

Comment (Joabutt) [reply README.md:1102]: if i dont have enough users whyd you add dee.al(https://domain.dee.al/), they literally have 2 users

Comment (Joabutt) [reply README.md:1102]: also arent they also using cloudflare

Comment (ripienaar) [reply README.md:1102]: As I mentioned, I didn't ask others these questions - but here we are, people just keep adding more, at some point this is enough of services adding no value.  That's now.

I've blocked @Joabutt from this repository.

**Comment by FatGrizzly** [comment]
Commenting on this, I feel its already misleading to call it "domains". These are subdomains, aren't listed in PSL(public suffix list either).

Though a few are listed though, like eu.org

**Comment by ripienaar** [comment]
I tend to allow them if the service permits NS records at a minimum

**Pull request closed without merging**